user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc

```diff
@@ -0,0 +1,189 @@
+
+----
+sed -i 's|main|f40|g' fedora-iot.conf
+----
+IMPORTANT: You will need to manually change the comps repo back to main. There are no branches in comps. (comps files are XML files used by various Fedora tools to perform grouping of packages into functional groups. For more information visit [click here](https://pagure.io/fedora-comps).
+
+Update the 'global_ksurl': 
+
+----
+sed -i 's|=HEAD|=origin/f40|g' fedora-iot.conf
+----
+
+Update the location of where the release is copied to during the compose process, and hosted for download and rsync'ing to the various Fedora mirrors. This path is created when the `twoweek-nightly.sh` is executed and can be found on the [Fedora Project master mirror](https://dl.fedoraproject.org/pub/alt/iot/).
+----
+sed -i 's|/pub/alt/iot/rawhide/|/pub/alt/iot/branched/|g' twoweek-nightly.sh
+----
+
+Commit the changes:
+
+----
+git commit -a -m "F-40: Update for branching" -s
+----
+
+Review the changes and if satisfied, push to your fork and open a pull request for others to review.
+
+== Working with the OStree repository
+
+This repository is used to configure the Fedora IoT ostree and includes the packages, services and various settings including selinux and unified-core.
+
+Clone and fork the upstream ostree repository:
+
+----
+git clone https://pagure.io/fedora-iot/ostree.git
+----
+
+Create the new branch and push to the upstream repository. Change `$release` to the new numeric branch of Fedora:
+
+----
+git checkout main; git pull; git checkout -b f$release; git push --set-upstream upstream f$release
+----
+
+Example, used for Fedora 40 (the next stable release of Fedora):
+
+----
+git checkout main; git pull; git checkout -b f40 ; git push --set-upstream upstream f40
+----
+
+=== Rawhide (main) 
+Create a branch in your local fork and make the following changes for Rawhide:
+
+----
+sed -i 's|40|41|g' config.ini fedora-40.repo fedora-iot-base.yaml fedora-iot.yaml
+mv fedora-40.repo fedora-41.repo
+git add fedora-41.repo
+git commit -a -m "IoT: Update rawhide for F-41" -s
+----
+
+Review the changes and if satisfied, push to your fork and open a pull request for others to review.
+
+=== Branched (the next stable fedora release)
+Checkout the newly created branch for the next stable Fedora and create a branch in your fork for the PR. 
+
+Change the urls from `development/rawhide` to `development/40`:
+
+----
+sed -i 's|development/rawhide|development/40|g' config.ini
+----
+
+Update instances of `rawhide`, replacing with `devel`:
+
+----
+sed -i 's|rawhide|devel|g' config.ini fedora-40.repo fedora-iot-base.yaml fedora-iot.yaml fedora-iot-updates-stable.yaml fedora-iot-updates-testing.yaml
+----
+
+Write the commit message:
+
+----
+git commit -a -m "Setup for F-40 branched" -s
+----
+
+Review the changes and if satisfied, push to your fork and open a pull request for others to review.
+
+== Additional Checks
+* check to make sure the Fedora IoT tag has been created in koji
```

Comment:
Where do I do this?  How can I confirm it was created?

Reply:
Fedora rel-eng does it as part of their SOP so we don't need to, we can check it in koji  (either webux or cli).

Reply:
> Where do I do this? How can I confirm it was created?

Clarified the package needed and updated the command.